Content.Client/Atmos/UI/GasAnalyzerWindow.xaml.cs

```diff
@@ -0,0 +1,197 @@
+using Content.Shared.Atmos;
+using Content.Shared.Temperature;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.XAML;
+using static Content.Shared.Atmos.Components.SharedGasAnalyzerComponent;
+
+namespace Content.Client.Atmos.UI
+{
+    [GenerateTypedNameReferences]
+    public sealed partial class GasAnalyzerWindow : DefaultWindow
+    {
+        public GasAnalyzerBoundUserInterface Owner { get; }
+        private TabContainer _tabContainer => CTabContainer;
+        private Button _refreshButton => RefreshButton;
+        private BoxContainer _topBox => CTopBox;
```

Comment:
you shouldn't need to have these, you can just refer to `CTabContainer` etc. if you need them so that the BUI can access them, you can change access to public in XAML